utils/reqApi.js

```diff
@@ -353,6 +382,34 @@ class ABRequestAPI {
       return true;
    }
 
+   /**
+    * @method validSwitcheroo()
+    * Verify if the current user has the Switcheroo Role assigned
+    * @param {bool} autoRespond
+    *        do we auto res.ab.error() on a negative result
+    *        see validUser() method.
+    * @return {bool}
+    */
+   validSwitcheroo(autoRespond = true) {
+      // these are the default Builder & System Designer Roles:
```

Comment:
Can you remove this comment? It confused me for a minute.

Reply:
done